litgpt/scripts/convert_hf_checkpoint.py

```diff
@@ -272,6 +290,24 @@ def copy_weights_phi(
             del qkv_weights[i][weight_type]
 
 
+def qkv_reassemble(param: Union[torch.Tensor, NotYetLoadedTensor], config: Config) -> torch.Tensor:
```

Comment:
This is great, I think this is maybe just what I needed for OLMo

Reply:
Hi @rasbt 
I was looking for OLMo and stumbled on your comment. Are there any plans to support the OLMo architecture in the near future? Couldn't find it here - https://github.com/Lightning-AI/litgpt/blob/main/litgpt/config.py